sql-cli/include/base/.airflow/.dags/.sql/example_templating/joint_orders_customers.sql

```diff
@@ -0,0 +1,4 @@
+SELECT c.customer_id, customer_name, order_id, purchase_date, amount, type
```

Comment:
Same here; shouldn't this be within `include/base/workflows`?

I'd expect for the folder `.airflow/dags/` to be empty by default

Reply:
I would suggest we have some example dags in the repo.